pkgs/top-level/cuda-packages.nix

```diff
@@ -108,7 +109,9 @@ let
     })
     (callPackage ../development/cuda-modules/cuda-samples/extension.nix {inherit cudaVersion;})
     (callPackage ../development/cuda-modules/cuda-library-samples/extension.nix {})
-  ];
+  ] ++ lib.optionals (config.allowAliases or true) [
```

Comment:
If it's defined in `top-level/config.nix` it's guaranteed to be an attribute?

Reply:
TIL, thanks
```
[13:39:55] jringer@jringer-5560-nixos ~/projects/nixpkgs (jringer/auto-add-driver-runpath)
$ nix repl
Welcome to Nix 2.18.1. Type :? for help.

nix-repl> :l
Added 20764 variables.

nix-repl> config.allowAliases
true
```

Reply:
applied